spec/lib/logstasher/instrumentation_spec.rb

```diff
@@ -65,5 +65,44 @@ def subject.index(*args)
         end
       end
     end
+
+    context "payload has custom fields defined" do
+      before :each do
```

Comment:
`:each` part is redundant. `before { some_code }` is same as `before(:each) { some_code }`. This also applies to `after` blocks